go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/operator-framework/operator-sdk
 go 1.13
 
 require (
+	github.com/MakeNowJust/heredoc v1.0.0 // indirect
```

Comment:
It doesn't look like there are any new dependencies added in this PR. Can you
```sh
$ git checkout origin/master -- go.mod go.sum
$ make install tidy
```
then push